package.json

```diff
@@ -8,8 +8,8 @@
     "url": "https://github.com/ourzora/v3.git"
   },
   "files": [
-    "./dist/**/*",
-    "./dist/*"
+    "dist/**/*",
+    "dist/*"
```

Comment:
@tbtstl this solved weird issue where gh action npm publish didnt include dist folder